staging/src/k8s.io/apimachinery/pkg/labels/selector.go

```diff
@@ -313,7 +313,7 @@ func (r Requirement) Equal(x Requirement) bool {
 	if r.operator != x.operator {
 		return false
 	}
-	return stringslices.Equal(r.strValues, x.strValues)
+	return slices.Equal(r.strValues, x.strValues)
```

Comment:
what's the performance difference between the specific implementation and the generic? are we planning to remove the stringslices package?

Reply:
We committed to supporting `k8s.io/utils` - but we could^Wshould deprecate these, or reimplement them in terms of `slices`, or BOTH